clicache/integration-test/PutGetTestsN.cs

```diff
@@ -318,9 +318,9 @@ public void DoRunQuery()
       }
       else
       {
-        QueryService<object, object> qs = null;
-        qs = CacheHelper.DCache.GetPoolManager().Find(m_region.Attributes.PoolName).GetQueryService<object, object>();
-        Query<object> qry = qs.NewQuery("SELECT * FROM " + m_region.FullPath);
+        QueryService qs = null;
+        qs = CacheHelper.DCache.GetPoolManager().Find(m_region.Attributes.PoolName).GetQueryService();
```

Comment:
Nit - clean things like this where declared on one line and assigned on the line below.